src/services/completions.ts

```diff
@@ -11,7 +11,7 @@ namespace ts.Completions {
     }
     export type Log = (message: string) => void;
 
-    const enum SymbolOriginInfoKind {
+    export const enum SymbolOriginInfoKind {

```

Comment:
If this is exported just for tests, should it be internal?

Reply:
The containing namespace declaration is already internal.

Reply:
Gets me every time. :smile: